app/controllers/web_hooks/slack_controller.rb

```diff
@@ -18,14 +18,20 @@ def create
     render plain: build_create_response
   end
 
+  # rubocop:disable Metrics/AbcSize
```

Comment:
Personally totally fine with disabling this here; could anyone with more Ruby experience shed some light on ways to structure code in a more `sqrt(A*A + B*B + C*D)` way?

Reply:
I had this same error in the `create` function when I was working on the slack controller like 2 weeks ago, which is why I created the separate `build_create_response` function and it was reduced. Also found that having if statements in one line helped but that's all the experience I have with it.

For this case, I could create another function `check_message_subtype` or something like that